package.json

```diff
@@ -55,9 +55,9 @@
     "karma-coverage-istanbul-reporter": "^3.0.3",
     "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "1.5.4",
-    "ng-packagr": "^15.0.3",
+    "ng-packagr": "16.2.3",
     "protractor": "~7.0.0",
```

Comment:
If there's no automated test suite on this library, I would get rid of protractor from the list of dependencies.